runtime/near-vm-runner/src/cache.rs

```diff
@@ -292,3 +292,60 @@ pub mod wasmer1_cache {
         }
     }
 }
+
+pub(crate) fn precompile_contract_impl(
+    vm_kind: VMKind,
+    wasm_code: &ContractCode,
+    config: &VMConfig,
+    cache: Option<&dyn CompiledContractCache>,
+) -> Result<bool, ContractPrecompilatonError> {
```

Comment:
What does `bool` indicate here? (well, I got the answer reading the code and the comment below) I would love to have an enum to easier reason about the return value

Reply:
Right, enum looks way better - fixed.